IPs/RSKIP531.md

```diff
@@ -0,0 +1,73 @@
+status: Draft
+purpose: Sec
+author: PDG (@patogallaiovlabs)
+layer: Core
+complexity: 2
+created: 2025-09-01
+---
+
+# Mitigation of Fee-Withholding Incentive in REMASC
+
+| RSKIP          | XXX                       |
+| :------------ |:-------------|
+| **Title**      | Mitigation of Fee-Withholding Incentive in REMASC |
+| **Created**    | 01-SEP-2025                                       |
+| **Author**     | Patricio Gallardo                         |
+| **Purpose**    | Sec                                         |
+| **Layer**      | Core                                              |
+| **Complexity** | 2                                                 |
+| **Status**     | Draft                                             |
+
+## Abstract
+
+This RSKIP proposes modifying the REMASC reward scheme to prevent fee-withholding strategies that degrade network performance. Strategic miners may exclude transaction fees to maximize their REMASC rewards, but this behavior reduces network throughput, increases confirmation delays, and creates congestion. We propose a defense mechanism: immediate partial fee distribution. A small fraction (e.g., 3%) of transaction fees would be instantly awarded to the current block producer, eliminating the incentive for fee withholding while preserving REMASC's security benefits.
+
+## Motivation
+
+REMASC introduces a delayed miner reward system to incentivize long-term honest behavior. However, its decoupling of fee inclusion and reward attribution creates an incentive for strategic miners to withhold transaction fees. This fee-withholding strategy, while potentially profitable for individual miners, actively degrades the network in several ways:
+
+* **Increased Confirmation Delays**: Strategic miners delay transaction inclusion to future blocks where they can profit from the fees, slowing down confirmations
+* **Economic Inefficiency**: Higher fees actually incentivize miners to delay inclusion further, creating a perverse incentive where users pay more for slower service
+* **Uneven Reward Distribution**: Strategic fee withholding creates an unfair advantage, favoring miners who adopt this behavior over those who don't
+* **No Penalization**: Excluded transactions simply roll over to future blocks, meaning there's no cost to miners for withholding fees
+
+We demonstrate both theoretically and empirically that this strategy yields an advantage for medium-hashrate strategic miners while degrading overall network performance. Addressing this vulnerability is critical to maintaining network efficiency and user experience.
+
+## Specification
+
+Introduce a protocol change where a configurable fraction $\alpha \in [0,1]$ of the transaction fees in each block is immediately credited to the miner of that block. The remaining $1 - \alpha$ continues to accumulate within REMASC and is distributed according to the existing delayed reward logic.
```

Comment:
Out of curiosity, I searched on github for "alpha" in rskj. The potential for conflict or confusion is practically nonexistent
 
The only other use of alpha appears in https://github.com/rsksmart/rskj/blob/master/rskj-core/src/main/java/org/ethereum/net/server/Stats.java